content/en/docs/examples/local-testing-with-kind/index.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Testing Istio Locally With Kubernetes In Docker
+description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation.
+weight: 10
+keywords:
+- kubernetes
+- kind
+- metallb
+- cert-manager
+- ingress
+---
+
+## About
+
+This example uses Kubernetes In Docker ([KIND][kind]) to deploy Istio locally. The cluster produced is intended to be used for further testing, evaluation and experimentation with Istio.
+
+## Deploying Kubernetes
+
+{{< tip >}}
+This section deploys a [Kind][kind] cluster similar to how the [Istio Kind Documentation](/docs/setup/platform-setup/kind) recommends, but with several important follow up steps to deploy certificate management and Istio gateways with TLS. Familiarize yourself with that documentation and [Kind][kind] itself to make sure you have a general working knowledge of `kind` before proceeding.
```

Comment:
If you check the [preview](https://deploy-preview-7155--preliminary-istio.netlify.app/docs/examples/local-testing-with-kind/) you'll see that some of the links (like this `[Kind][kind]` one) are not displaying correctly. I think the problem may be that the square bracket syntax doesn't work inside tips (although it does seem to work fine outside of them). So anyway, I think you'll need to use the expanded form `[title](target link)` in the places where it doesn't work.

That said, for this particular case, I don't think you should be making every use of kind a clickable link. Just the first occurrence would be enough. After that, just enclose it in backticks: `kind`.

Reply:
Sounds good, I made those changes as suggested :+1: